src/jdk.compiler/share/classes/com/sun/tools/javac/code/Types.java

```diff
@@ -5043,8 +5043,7 @@ public boolean checkUnconditionallyExact(Type source, Type target) {
         }
 
         if (target.isPrimitive()) {
```

Comment:
nit: maybe now a ternary expression can be used

Reply:
Done!